cumulusci/tasks/salesforce/users/permsets.py

```diff
@@ -120,9 +123,40 @@ def _get_assignments(self, user_id, assigned_perms):
         return assignments
 
     def _insert_assignments(self, records_to_insert):
-        request_body = json.dumps({"allOrNone": False, "records": records_to_insert})
-        result = self.sf.restful("composite/sobjects", method="POST", data=request_body)
-        self.logger.debug(result)
+        result_list = []
+        for i in range(0, len(records_to_insert), 200):
+            request_body = json.dumps(
+                {"allOrNone": False, "records": records_to_insert[i : i + 200]}
+            )
+            result = self.sf.restful(
+                "composite/sobjects", method="POST", data=request_body
+            )
+            self.logger.debug(result)
```

Comment:
With the table, can we now avoid logging the raw JSON?

Reply:
Generally, I like having API responses when debugging, but you're right that it's duplicative here.